### DISCUSSION TITLE
How to stop SEB running inside Vmware workstation even I change the .vmx conf file settings as "SMBIOS.reflectHost = "TRUE""

### DISCUSSION THREAD

**Discussion by Chris251290** [SafeExamBrowser/seb-win-refactoring#454] [Q&A] [answered]
Hi Team,

We have created SEB File which is not allowed to run SEB inside vmware workstation. If candidates modified **.vmx configuration** file and added the line "**SMBIOS.reflectHost = "TRUE"**" the virtual machine will act as a normal machine and can able to run SEB inside the vmware without any issues. 

So, How to block SEB running inside vmware workstation and all other virtual workstation applications? 

Is there any other services to be prohibit inside the vmware to stop running SEB?

Kindly help in this.

**Comment by danschlet** [comment, marked as answer]
This trick was mentioned in previous issues (please search existing issues before posting) and shouldn't work anymore in SEB 3.4. Did you try this latest SEB version?

**Reply by Chris251290** [reply]
Hi Dan,

Thank you. Yes. I have tried SEB 3.4. It is detecting Virtual machine and also if we change the virtual machine as normal host. Now the SEB is not running inside the VM Virtual machine. 

Great!!! Thank you.
